src/tests/ftest/util/telemetry_test_base.py

```diff
@@ -265,3 +265,27 @@ def sum_values(metric_out):
                     total += value
 
         return total
+
+    def get_aggregate_metrics(self, metrics=None):
```

Comment:
It looks like the test is now using `GX` object classes, so do we still want to keep this (unused) function?